infra/op-replica/docker-compose/replica.yml

```diff
@@ -15,7 +15,7 @@ services:
       - ${SHARED_ENV_PATH}/data-transport-layer.env
       - .env
     volumes:
-      - dtl:/db
+      - /var/replica/dtl:/db
```

Comment:
I'm fine adding README on how to preserve the data from the replica.
But, this change will clobber whatever the user has on their locale system at the `/var/replica` path.

If you feel strongly that we should default to host paths instead of docker volumes, the paths should be set in the `.env` file for the user to review before it makes changes to their system.